src/acl/Acl.cc

```diff
@@ -306,13 +307,13 @@ ACL::ParseAclLine(ConfigParser &parser, ACL ** head)
 }
 
 bool
-ACL::isProxyAuth() const
+Acl::ACL::isProxyAuth() const
 {
     return false;
 }
 
 void
-ACL::parseFlags()
+Acl::ACL::parseFlags()
 {
     Acl::Options allOptions = options();
     for (const auto lineOption: lineOptions()) {
```

Comment:
Please remove `Acl::` namespace prefixes from the old code that this PR moves inside that Acl namespace, like the allOptions declaration line above. This change request applies to all such occurrences.

Reply:
done as far as I can tell in latest code.